### ISSUE TITLE
【bug】 $(npm bin)/artillery dino failed

### ISSUE THREAD

**Issue by magentaqin** [artilleryio/artillery#1411]
Version info: 
```
 2.0.0-14
```

Running this command: 
```
$(npm bin)/artillery dino failed
```

I expected to see this happen:

![Screen Shot 2022-04-19 at 5 01 49 PM](https://user-images.githubusercontent.com/30370223/163969252-9554a770-ace9-4c5e-9bb3-bd3311189c5e.png)

Instead, this happened:

![Screen Shot 2022-04-19 at 4 59 15 PM](https://user-images.githubusercontent.com/30370223/163969137-e181d85c-9a21-424a-bce5-74103dce24fa.png)

*explanation*：
lib/console-reporter.js accidentally has optional chaining operator, which can not be understood by node.

Files being used:
```
<relevant yaml/js/csv go here>
```

**Comment by hassy** [comment]
hi @magentaqin 👋 

Artillery depends on Node.js v14 or higher. I'm guessing you're running an older version of Node which does not support optional chaining.

https://github.com/artilleryio/artillery/blob/master/package.json#L7

**Comment by magentaqin** [comment]
Thanks. After I upgrade my node version, it works!

**Comment by YassinEldeeb** [comment]
@hassy probably introducing a build system would allow for compiling to an older target so for example the optional chaining would be unwinded to use if statements and this sort of stuff and then It would support much older Node.js versions.

**Comment by hassy** [comment]
Supporting older Node.js versions isn't really something we want to do. We try to match current LTS or latest AWS Lambda runtime version, whichever is newer.

A smaller release bundle could be interesting if it significantly reduces package size or CLI start time for example.

**Comment by YassinEldeeb** [comment]
> Supporting older Node.js versions isn't really something we want to do

Yes you're completely right, we're now in Node.js 18! People using super older versions should have a problem.

> A smaller release bundle could be interesting if it significantly reduces package size or CLI start time for example.

I think it can significantly reduce package size but not the CLI start time, cause It just transforms variable names to be shorter, removes unnecessary spaces, line breaks, comments, ..etc. basically removes everything which won't affect node's runtime.

**Comment by magentaqin** [comment]
Runtime compatible code is unavoidable which could pump up the size. A waring "Please upgrade to higher node version" instead of an error would be more friendly to users. I will pull a request to optimize it.

**Comment by YassinEldeeb** [comment]
I agree 💯
